src/views/AfhandelingView.vue

```diff
@@ -456,6 +444,28 @@ onMounted(() => {
     if (!vraag.kanaal) {
       vraag.kanaal = userStore.preferences.kanaal;
     }
+
+    if (!vraag.vraag) {
+      vraagOptions.value = [];
+      continue;
+    }
+    const sectionIndex = vraag.vraag.sectionIndex;
+
+     vraagOptions.value = computed(() => [
+      ...vraag.websites.map((item) => item.website),
+      ...vraag.kennisartikelen.flatMap((item) => [
+        item.kennisartikel,
+        ...item.kennisartikel.sections.map((section) => ({
+          ...item.kennisartikel,
+          title: [item.kennisartikel.title, section].join(' - '),
+        })),
+      ]),
+      ...vraag.nieuwsberichten.map((item) => item.nieuwsbericht),
+      ...vraag.werkinstructies.map((item) => item.werkinstructie),
+      ...vraag.vacs.map((item) => item.vac),
+    ]).value;
+
+    vraag.vraag = vraagOptions.value[sectionIndex];
```

Comment:
Heb je dit getest met een contactmoment met meerdere vragen? Lijkt me niet wenselijk dat de opties voor alle vragen hetzelfde zijn. En hoe werkt dit voor items die geen kennisartikel zijn, en dus ook geen sectionIndex hebben?

Reply:
goed gespot, heb deze nu goed door getest en hij selecteert nu altijd de juiste sectie ongeacht hoeveel kennisartikelen of VACs etc. er zijn

Reply:
refactored, comment is outdated.

alles is nu opnieuw getest en werkt naar behoren